angular-client/src/components/graph-caption/graph-caption.component.css

```diff
@@ -0,0 +1,23 @@
+.info-container {
+  font: small-caption;
+  position: absolute;
+  width: 80%;
```

Comment:
this doesnt work, you need to set the width to be the parents width - 200px i think, if you shrink the screen it starts scrolling